src/web/app/.dockerignore

```diff
@@ -4,6 +4,7 @@ npm-debug.log
 Dockerfile
 .git
 .gitignore
-
 .next
 out
+ .env*
+ env*
```

Comment:
Do we have `.env` in this directory? Not sure if this is as a pre-caution?

Reply:
Yes, it's a precaution

Reply:
I really wonder if we should include it then... I can't find anything that suggests which is better/not?